src/Workspaces/Core/Portable/Options/IGlobalOptionService.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.CodeAnalysis.Options
     /// all workspaces/services).
     /// 
     /// In general you should not import this type directly, and should instead get an
-    /// <see cref="IOptionService"/> from <see cref="Workspace.Services"/>
+    /// <see cref="ILegacyWorkspaceOptionService"/> from <see cref="Workspace.Services"/>
```

Comment:
hmm - seems weird to suggest that we use a legacy service.  Is this still the current recommended way?

Reply:
This comment is stale. Will delete. `IGlobalOptionService` should be used directly.